File: cs201/midterm-programming-spring2017-SOLUTION.zip_expanded/CSCI201Factory_PE1/src/client/ManagerOffice.java
package client;

//PE1
//see Lab 2: FactoryWall.java for reference
public class ManagerOffice extends FactoryObject {
	
	private static final long serialVersionUID = 1L;

	//set ManagerOffice name, image, and location using FactoryObject constructor
	public ManagerOffice(int x, int y) {
		super("Manager Office", "ManagerOffice" + Constants.png, x, y);
	}

	public String getName() {
		return name;
	}
}

File: cs201/midterm-programming-spring2017-SOLUTION.zip_expanded/CSCI201Factory_PE1/WebContent/js/ManagerOffice.js
//PE1
// creates a ManagerOffice with the name and image data from Factory's managerOffice data member
//See Lab 2 -> FactoryWall.js for reference
function ManagerOffice(cell, managerOffice) {
	// create and place image
	var name = managerOffice['name'];
	var img = document.createElement('img');
	img.id = name;
	img.src = 'img/' + managerOffice.image;
	//append it to the bottom right grid cell
	cell.appendChild(img);
}